### PR TITLE
C3 fattouh92 s22 import csv ro

### PR DESCRIPTION
**Pull request by abulazm22**
#186

@m0hamed @Mirna-Yacout @SalmaFG @nohahesham

### CODE REVIEW

**Comment by Mirna-Yacout** [comment]
- why do u start from the sign in? why dont u just start your UML from import CSV since this is the story
- there should be a box on the timeline for import_csv view since there is more than one arrow
- conditions should be in square brackets
- u forgot the if condition for !temp in the else in project controller
- lines 274 till 282 are not in the UML
- leave space between inner blocks in last opt block
- general note: blocks should start from the beginning of an arrow

**Comment by salmafg** [comment]
Testing verified.

File: arability/app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery
+  # rescue_from Exception, :with => :error_render_method
   before_filter :set_locale
```
Comment (nohahesham) [inline arability/app/controllers/application_controller.rb:4]: remove commented lines

File: arability/app/controllers/projects_controller.rb [hunk 2]
```diff
@@ -30,32 +30,20 @@ def destroy
...
-        redirect_to developers_new_path
-      end
-    else
-      flash[:error] = t(:projects_index_error2)
-      redirect_to new_gamer_session_path      
-    end
+    @developer = Developer.where(:gamer_id => current_gamer.id).first
+    @my_projects = Project.where(:owner_id => @developer.id)
+    @shared_projects = @developer.projects_shared
   end
```
Comment (nohahesham) [inline arability/app/controllers/projects_controller.rb:46]: dont use =>

File: arability/app/views/projects/index.html.erb [hunk 3]
```diff
@@ -10,123 +5,194 @@
...
+                    <% if I18n.locale == :en %>
+                      Formal
+                    <% else %>
+                      فصحى
+                    <% end %>
+                  <% else %>
+                    <% if I18n.locale == :en  %>
+                      Slang
+                    <% else %>
+                      عامية
```
Comment (nohahesham) [inline arability/app/views/projects/index.html.erb:49]: can u use the translations?

Comment (abulazm22) [inline arability/app/views/projects/index.html.erb:49]: Noha this is from another story

**Comment by Mirna-Yacout** [comment]
verified

**Pull request merged**